ompi/mca/pml/yalla/pml_yalla.c

```diff
@@ -716,8 +717,12 @@ int mca_pml_yalla_start(size_t count, ompi_request_t** requests)
                 }
             }
         } else {
+            mca_pml_yalla_recv_request_t *rreq;
+            rreq = (mca_pml_yalla_recv_request_t *)req;
```

Comment:
@artpol84 @yosefe OMPI is cool with declarations in the middle of blocks (or sub-blocks) -- OMPI requires a C99 compiler, so it's guaranteed to be portable.